lib/li_matrix.c

```diff
@@ -660,11 +660,20 @@ li_frustum_spherical_from_cartesian(LIFRUSTUM *res, const LIFRUSTUM *f)
 }
 
 
+void
+li_distortion_set(LIDISTORSION* dist, double pps0, double pps1, double c0, double c1, double c2)
```

Comment:
This function should set the `r2max` member of `dist`, which defines the definition domain of the transform using a squared distance from the `pps` point. Otherwise the undefined value is likely to result in `PC_FAILURE` returns.

Options are :
1. (simpler) initialize `r2max=0` to explicitly disable the check for domain validity when no value is provided : [here](/LI3DS/pointcloud/blob/bc138adc112c0ad4ea2f2efa3c4fe425561a678b/lib/li_matrix.c#L683), [here](/LI3DS/pointcloud/blob/bc138adc112c0ad4ea2f2efa3c4fe425561a678b/lib/li_matrix.c#L736) and [here](/LI3DS/pointcloud/blob/bc138adc112c0ad4ea2f2efa3c4fe425561a678b/lib/li_matrix.c#L799)
2. (preferred) compute the `r2max` here if it is not provided or provided with a null/0 value. relevant code can be found in [/IGNF/libOri](/IGNF/libOri) : [here](/IGNF/libOri/blob/e5b0234bf52cae34401df465d8ae09ce348f969b/src/DistortionPolynom.cpp#L142]) and [here](/IGNF/libOri/blob/e5b0234bf52cae34401df465d8ae09ce348f969b/src/DistortionPolynom.cpp#L11-L84)
3. (both) have `li_distortion_set` input support `r2max=0` for disabling domain check, and `r2max<0` for automatic computation.